src/components/pager/pager.styles.js

```diff
@@ -90,10 +99,11 @@ const PagerLinkStyles = styled.button`
   &:focus {
     outline: none;
   }
+  color: rgba(0,0,0,0.90);
 
   ${({ isDisabled }) => isDisabled && css`
-    color: rgba(0,0,0,0.3);
-  `};
+    color: rgba(0,0,0,0.55);
```

Comment:
don't we have this color in a config?

Reply:
Not as far as I can see, I'll add them to the config though so the colours aren't hardcoded